app/src/androidTest/java/com/github/multimatum_team/multimatum/MainActivityTest.kt

```diff
@@ -181,6 +186,46 @@ class MainActivityTest {
         }
     }
 
+    /*
+    The two following helper functions have been found online at:
+    https://alexzh.com/ui-testing-of-android-runtime-permissions/
+     */
+    private fun grantPermission() {
+        val instrumentation = InstrumentationRegistry.getInstrumentation()
+        if (Build.VERSION.SDK_INT >= 23) {
```

Comment:
I think that this check is useless. In `app/build.gradle`, `minSdk` is set to 26.

Reply:
Yes correct